grouper/fe/settings.py

```diff
@@ -25,6 +25,7 @@ def override_timezone(self, value):
     "permission_request_text_help": None,
     "port": 8989,
     "refresh_interval": 60,
+    "shell": [["/bin/bash", "bash"], ["/bin/zsh", "zsh"]],
```

Comment:
i see. you can use patch in a context instead which works around this. also, fwiw, you should [patch at the point of import](http://www.voidspace.org.uk/python/mock/patch.html#where-to-patch) vs where it's defined.

so something like
```
def test_shell(...):
  with('grouper.fe.handlers.user_shell.settings') as mock_settings:
    mock_settings.settings.shell = [['blha', 'blah'], ['foo', 'foo']]
    [...]
```